dev-db/postgresql/postgresql-11.20.ebuild

```diff
@@ -386,7 +386,14 @@ pkg_config() {
 	einfo "Initializing the database ..."
 
 	if [[ ${EUID} == 0 ]] ; then
-		su - postgres -c "${EROOT}/usr/$(get_libdir)/postgresql-${SLOT}/bin/initdb -D \"${DATA_DIR}\" ${PG_INITDB_OPTS}"
+		local cmd="${EROOT}/usr/$(get_libdir)/postgresql-${SLOT}/bin/initdb -D \"${DATA_DIR}\" ${PG_INITDB_OPTS}"
```

Comment:
Move the variable declaration before the `if` statement, and we can use it in the `else` branch. Also, a more descriptive name would be better. How about `initdb_cmd`?

Reply:
The command is slightly different in the else branch. See `-U postgres`.